### ISSUE TITLE
Can't enable driver, option to allow in "Security & Privacy" never appears

### ISSUE THREAD

**Issue by maxrothman** [360Controller/360Controller#989]
### Type of Controller
Xbox 360

### OS Version
10.14.6, Macbook Pro (Retina, 15", mid-2015)

### Driver Version
1.0.0-alpha.5

### Connection Method
Wired

### Device Name and Info
>Device Name: Controller
<br/>Product ID: 0x028e
<br/>Vendor ID: 0x045e  (Microsoft Corporation)

### Response of Controller
Green ring lights up once then goes dark

### Response in Preference Pane
Device list says "No devices found"

### Detailed Explaination of Issue
I am unable to activate the driver. Here are the steps I took:
* Disable any potentially conflicting software, including Karabiner, BetterTouchTool, Alfred, etc.
* Reinstall driver (and restart the computer)
* Plug in controller
* Check "Enable driver" in the pref pane
* Click button switching view to "Security & Privacy"

Result: the button allowing me to allow the developer "Drew Mills" does not appear

Here's the output of `sudo kextutil /Library/Extensions/360Controller.kext`:
```
Error copying bundle during staging: -1
Unable to stage kext (/System/Library/Extensions/roller.kext) to secure location.
Notice: /Library/Extensions/360Controller.kext has debug properties set.
Kext rejected due to system policy: <OSKext 0x7fd40dd98500 [0x7fffab9238e0]> { URL = "file:///Library/StagedExtensions/Library/Extensions/360Controller.kext/", ID = "com.mice.driver.Xbox360Controller" }
Kext rejected due to system policy: <OSKext 0x7fd40dd98500 [0x7fffab9238e0]> { URL = "file:///Library/StagedExtensions/Library/Extensions/360Controller.kext/", ID = "com.mice.driver.Xbox360Controller" }
Diagnostics for /Library/Extensions/360Controller.kext:
Warnings:
    Personality CFBundleIdentifier differs from containing kext's (not necessarily a mistake, but rarely done):
        ChatPadKeyboardEvents
```
Is there some way I can use `kextutil` to accept the developer certificate, or are there logs I can view to find out why the button doesn't appear?

**Comment by FranticRain** [comment]
Every time you run `kextuil`, it should make the prompt re-appear within the "Security & Privacy" section of the preference pane. It will be at the bottom of the "General" tab. Give it a shot and let me know if it works for you.

**Comment by maxrothman** [comment]
Nope, that had no effect.

**Comment by FranticRain** [comment]
I just want to make sure that the kext is signed correctly. It should be, but if that prompt isn't showing up, I'm inclined to believe that this is a notarization error. Run this command:
```
spctl -a -v --type install /Library/Extensions/360Controller.kext 
```
It should give you this:
```
/Library/Extensions/360Controller.kext: accepted
source=Notarized Developer ID
```

Also if you're using alpha 5, then you should be able to check the "enable driver" checkbox in the "Advanced" settings of the driver. This will run a diagnostic that will probably tell you that you need to accept the signing certificate, but I want to make sure. It gives a slightly more granular error than `kextutil` does. So check that and let me know what it says as well.

**Comment by maxrothman** [comment]
```shell
$ spctl -a -v --type install /Library/Extensions/360Controller.kext
/Library/Extensions/360Controller.kext: accepted
source=Notarized Developer ID
```
I am using alpha 5. As I described in my first message, when I originally checked the "enable driver" checkbox, it prompted me to switch to "Security & Privacy" and accept the certificate, where no accept button appeared. Now though, when I check the "enable" box System Preferences becomes unresponsive and the mouse switches to the beach ball of doom when it's over the app until I force-quit it.

**Comment by FranticRain** [comment]
This sounds like it may be an issue with your mac, specifically. We can try and make sure that the driver/preference pane has no bearing on this. Go ahead and go through the process where it jumps you to the security pane. Then force quit out of system preferences, open it back up to that pane and click the button. It *should* still be there. if it still crashes, then your Mac is probably to blame in some way.

Of note is that devices and software that mess with inputs can break this button. Notably things like Wacom tablets and Karabiner Elements are known troublemakers. If you have any software or devices that do this sort of input editing, you might want to disable them, or unplug them, to see if they are causing this issue.

**Comment by Moorviper** [comment]
click enable the driver and it should work.
Then at security allow the two extensions / the message that they have been blocked is there.
I have the same Macbook and 10.14.6 installed
at least it does for me.

<img width="874" alt="Bildschirmfoto 2019-12-16 um 11 15 34" src="https://user-images.githubusercontent.com/3305254/70898694-7181c500-1ff5-11ea-8dac-bedb9bb71e1f.png">
<img width="874" alt="Bildschirmfoto 2019-12-16 um 11 10 36" src="https://user-images.githubusercontent.com/3305254/70898695-721a5b80-1ff5-11ea-87a3-c0e5c2e23ad1.png">

**Comment by gregrahg** [comment]
I am having the exact same issue as described the original post. Was this issue ever resolved?

**Comment by clivend** [comment]
same here :(

**Comment by clivend** [comment]
I am using alpha 5 on macos 10.15.3, when I click Enable Driver I get
<img width="432" alt="image" src="https://user-images.githubusercontent.com/2529599/77154938-ff5dc800-6a9c-11ea-9b83-c36598fadd2f.png">
then I open security preferences but all I see is
<img width="642" alt="image" src="https://user-images.githubusercontent.com/2529599/77154985-17cde280-6a9d-11ea-97d8-42febac68df3.png">
running `sudo kextutil /Library/Extensions/360Controller.kext`
gives
```
Notice: /Library/Extensions/360Controller.kext has debug properties set.
Kext rejected due to system policy: <OSKext 0x7ffc59e0acc0 [0x7fff95100690]> { URL = "file:///Library/StagedExtensions/Library/Extensions/360Controller.kext/", ID = "com.mice.driver.Xbox360Controller" }
Kext rejected due to system policy: <OSKext 0x7ffc59e0acc0 [0x7fff95100690]> { URL = "file:///Library/StagedExtensions/Library/Extensions/360Controller.kext/", ID = "com.mice.driver.Xbox360Controller" }
Diagnostics for /Library/Extensions/360Controller.kext:
Warnings:
    Personality CFBundleIdentifier differs from containing kext's (not necessarily a mistake, but rarely done):
        ChatPadKeyboardEvents
```

would be really cool to have it working during this strange times locked at home, thank you!

**Comment by FranticRain** [comment]
@clivend Do you have any software that modifies your inputs installed? Something like Karabiner? Or do you have a Wacom tablet installed? These types of devices can interfere with macOS’s willingness to let you approve security stuff.

**Comment by clivend** [comment]
@FranticRain thank you for your reply! I tried to disconnect everything but had no luck :(

here is a list of my software currently installed in case that helps. nothing really modifies input to my knowledge

```[
    {
        "name": "Calendar",
        "version": "11.0"
    },
    {
        "name": "System Preferences",
        "version": "14.0"
    },
    {
        "name": "Preview",
        "version": "11.0"
    },
    {
        "name": "Activity Monitor",
        "version": "10.14"
    },
    {
        "name": "Installer",
        "version": "6.2.0"
    },
    {
        "name": "DiskImageMounter",
        "version": "559.40.3"
    },
    {
        "name": "App Store",
        "version": "3.0"
    },
    {
        "name": "ColorSync Utility",
        "version": "4.14.0"
    },
    {
        "name": "System Information",
        "version": "10.14"
    },
    {
        "name": "Calculator",
        "version": "10.14"
    },
    {
        "name": "AirPort Utility",
        "version": "6.3.9"
    },
    {
        "name": "AirDrop",
        "version": "1.0"
    },
    {
        "name": "Audio MIDI Setup",
        "version": "3.5"
    },
    {
        "name": "Archive Utility",
        "version": "10.15"
    },
    {
        "name": "Stocks",
        "version": "2.3"
    },
    {
        "name": "Photos",
        "version": "5.0"
    },
    {
        "name": "Photo Booth",
        "version": "11.0"
    },
    {
        "name": "Folder Actions Setup",
        "version": "1.2"
    },
    {
        "name": "Font Book",
        "version": "10.0"
    },
    {
        "name": "About This Mac",
        "version": "1.0"
    },
    {
        "name": "Storage Management",
        "version": "1.0"
    },
    {
        "name": "Disk Utility",
        "version": "19.0"
    },
    {
        "name": "System Events",
        "version": "1.3.6"
    },
    {
        "name": "PIPAgent",
        "version": "1.0"
    },
    {
        "name": "KeyboardSetupAssistant",
        "version": "10.7"
    },
    {
        "name": "Certificate Assistant",
        "version": "5.0"
    },
    {
        "name": "Stocks Widget",
        "version": "1.0"
    },
    {
        "name": "Photo Library Migration Utility",
        "version": "2.0"
    },
    {
        "name": "LocationMenu",
        "version": "1.0"
    },
    {
        "name": "NowPlayingWidgetContainer",
        "version": "1.0"
    },
    {
        "name": "Language Chooser",
        "version": "1.0"
    },
    {
        "name": "AddPrinter",
        "version": "15"
    },
    {
        "name": "Spotlight",
        "version": "1.0"
    },
    {
        "name": "Captive Network Assistant",
        "version": "5.0"
    },
    {
        "name": "Setup Assistant",
        "version": "10.10"
    },
    {
        "name": "OSDUIHelper",
        "version": "1.0"
    },
    {
        "name": "RegisterPluginIMApp",
        "version": "20"
    },
    {
        "name": "Installer Progress",
        "version": "1.0"
    },
    {
        "name": "ScreenSaverEngine",
        "version": "5.0"
    },
    {
        "name": "CoreServicesUIAgent",
        "version": "340.1"
    },
    {
        "name": "PodcastsAuthAgent",
        "version": "1.0"
    },
    {
        "name": "SystemUIServer",
        "version": "1.7"
    },
    {
        "name": "NowPlayingTouchUI",
        "version": "1.0"
    },
    {
        "name": "Bluetooth Setup Assistant",
        "version": "7.0.3"
    },
    {
        "name": "ARDAgent",
        "version": "3.9.8"
    },
    {
        "name": "Siri",
        "version": "200.2.3"
    },
    {
        "name": "Dock",
        "version": "1.8"
    },
    {
        "name": "loginwindow",
        "version": "9.0"
    },
    {
        "name": "Pass Viewer",
        "version": "1.0"
    },
    {
        "name": "Expansion Slot Utility",
        "version": "2.0"
    },
    {
        "name": "screencaptureui",
        "version": "1.0"
    },
    {
        "name": "Software Update",
        "version": "6"
    },
    {
        "name": "ManagedClient",
        "version": "12.1"
    },
    {
        "name": "Install Command Line Developer Tools",
        "version": "2373"
    },
    {
        "name": "ScriptMonitor",
        "version": "1.0.1"
    },
    {
        "name": "EscrowSecurityAlert",
        "version": "1.0"
    },
    {
        "name": "CoreLocationAgent",
        "version": "2389.0.8"
    },
    {
        "name": "UniversalAccessControl",
        "version": "7.0"
    },
    {
        "name": "ControlStrip",
        "version": "1.0"
    },
    {
        "name": "AVB Audio Configuration",
        "version": "800.17"
    },
    {
        "name": "UnmountAssistantAgent",
        "version": "5.0"
    },
    {
        "name": "AirPort Base Station Agent",
        "version": "2.2.1"
    },
    {
        "name": "KeyboardAccessAgent",
        "version": "1.0"
    },
    {
        "name": "Games",
        "version": "1.0"
    },
    {
        "name": "BluetoothUIServer",
        "version": "7.0.3"
    },
    {
        "name": "Notification Centre",
        "version": "1.0"
    },
    {
        "name": "Weather",
        "version": "1.0"
    },
    {
        "name": "Memory Slot Utility",
        "version": "1.5.3"
    },
    {
        "name": "Problem Reporter",
        "version": "10.13"
    },
    {
        "name": "HindiIM",
        "version": "1.0"
    },
    {
        "name": "TCIM",
        "version": "1.0"
    },
    {
        "name": "PressAndHold",
        "version": "1.0"
    },
    {
        "name": "JapaneseIM",
        "version": "1.0"
    },
    {
        "name": "TrackpadIM",
        "version": "1.0"
    },
    {
        "name": "TYIM",
        "version": "1.0"
    },
    {
        "name": "PluginIM",
        "version": "20"
    },
    {
        "name": "50onPaletteServer",
        "version": "1.1.0"
    },
    {
        "name": "EmojiFunctionRowIM",
        "version": "1.0"
    },
    {
        "name": "CharacterPalette",
        "version": "2.0.1"
    },
    {
        "name": "TamilIM",
        "version": "1.6"
    },
    {
        "name": "AinuIM",
        "version": "1.0"
    },
    {
        "name": "VietnameseIM",
        "version": "1.0"
    },
    {
        "name": "KoreanIM",
        "version": "1.0"
    },
    {
        "name": "Assistive Control",
        "version": "2.0"
    },
    {
        "name": "SCIM",
        "version": "1.0"
    },
    {
        "name": "SmartCard Pairing",
        "version": "1.0"
    },
    {
        "name": "qlmanage",
        "version": "1.0"
    },
    {
        "name": "quicklookd",
        "version": "5.0"
    },
    {
        "name": "VoiceOver Quickstart",
        "version": "10"
    },
    {
        "name": "syncuid",
        "version": "8.1"
    },
    {
        "name": "Finder",
        "version": "10.15.2"
    },
    {
        "name": "Directory Utility",
        "version": "6.0"
    },
    {
        "name": "Mail",
        "version": "13.0"
    },
    {
        "name": "QuickTime Player",
        "version": "10.5"
    },
    {
        "name": "Podcasts",
        "version": "1.0.0"
    },
    {
        "name": "Reminders",
        "version": "7.0"
    },
    {
        "name": "Automator",
        "version": "2.10"
    },
    {
        "name": "Console",
        "version": "1.1"
    },
    {
        "name": "VoiceOver Utility",
        "version": "10"
    },
    {
        "name": "Bluetooth File Exchange",
        "version": "7.0.3"
    },
    {
        "name": "Script Editor",
        "version": "2.11"
    },
    {
        "name": "Keychain Access",
        "version": "10.5"
    },
    {
        "name": "Grapher",
        "version": "2.7"
    },
    {
        "name": "Launchpad",
        "version": "1.0"
    },
    {
        "name": "Mission Control",
        "version": "1.2"
    },
    {
        "name": "Voice Memos",
        "version": "2.1"
    },
    {
        "name": "Image Capture",
        "version": "8.0"
    },
    {
        "name": "FaceTime",
        "version": "5.0"
    },
    {
        "name": "Find My",
        "version": "1.0"
    },
    {
        "name": "TV",
        "version": "1.0.3"
    },
    {
        "name": "Maps",
        "version": "2.1"
    },
    {
        "name": "Notes",
        "version": "4.7"
    },
    {
        "name": "Contacts",
        "version": "12.0"
    },
    {
        "name": "Messages",
        "version": "13.0"
    },
    {
        "name": "Books",
        "version": "2.2"
    },
    {
        "name": "Stickies",
        "version": "10.2"
    },
    {
        "name": "Chess",
        "version": "3.16"
    },
    {
        "name": "Music",
        "version": "1.0.3"
    },
    {
        "name": "HelpViewer",
        "version": "6.0"
    },
    {
        "name": "Dictionary",
        "version": "2.3.0"
    },
    {
        "name": "Recents",
        "version": "1.0"
    },
    {
        "name": "iCloud Drive",
        "version": "1.0"
    },
    {
        "name": "Computer",
        "version": "1.0"
    },
    {
        "name": "Network",
        "version": "1.0"
    },
    {
        "name": "Recents",
        "version": "1.0"
    },
    {
        "name": "TextEdit",
        "version": "1.15"
    },
    {
        "name": "Siri",
        "version": "1.0"
    },
    {
        "name": "Digital Colour Meter",
        "version": "5.15"
    },
    {
        "name": "Boot Camp Assistant",
        "version": "6.1.0"
    },
    {
        "name": "Screenshot",
        "version": "1.0"
    },
    {
        "name": "Terminal",
        "version": "2.10"
    },
    {
        "name": "Migration Assistant",
        "version": "10.15"
    },
    {
        "name": "Home",
        "version": "4.0"
    },
    {
        "name": "Time Machine",
        "version": "1.3"
    },
    {
        "name": "Ticket Viewer",
        "version": "4.1"
    },
    {
        "name": "Feedback Assistant",
        "version": "4.6"
    },
    {
        "name": "Wireless Diagnostics",
        "version": "1.0"
    },
    {
        "name": "Screen Sharing",
        "version": "1.7.2"
    },
    {
        "name": "DVD Player",
        "version": "6.0"
    },
    {
        "name": "Network Utility",
        "version": "1.9.2"
    },
    {
        "name": "Display Calibrator",
        "version": "4.11.0"
    },
    {
        "name": "iCloud Drive",
        "version": "1.0"
    },
    {
        "name": "Speech Downloader",
        "version": "9.0.15"
    },
    {
        "name": "SpeechRecognitionServer",
        "version": "9.0.15"
    },
    {
        "name": "AskPermissionUI",
        "version": "1.0"
    },
    {
        "name": "CIMFindInputCodeTool",
        "version": "104"
    },
    {
        "name": "nbagent",
        "version": "1.0"
    },
    {
        "name": "AccessibilityVisualsAgent",
        "version": "1.0"
    },
    {
        "name": "IMAutomaticHistoryDeletionAgent",
        "version": "10.0"
    },
    {
        "name": "STMUIHelper",
        "version": "1.0"
    },
    {
        "name": "UASharedPasteboardProgressUI",
        "version": "54.1"
    },
    {
        "name": "Calibration Assistant",
        "version": "1.0"
    },
    {
        "name": "AXVisualSupportAgent",
        "version": "1.0"
    },
    {
        "name": "universalAccessAuthWarn",
        "version": "1.0"
    },
    {
        "name": "UniversalAccessHUD",
        "version": "1.0"
    },
    {
        "name": "DFRHUD",
        "version": "1.0"
    },
    {
        "name": "ParentalControls",
        "version": "4.1"
    },
    {
        "name": "storeuid",
        "version": "1.0"
    },
    {
        "name": "eaptlstrust",
        "version": "13.0"
    },
    {
        "name": "Conflict Resolver",
        "version": "8.1"
    },
    {
        "name": "IMTransferAgent",
        "version": "10.0"
    },
    {
        "name": "AOSPushRelay",
        "version": "1.07"
    },
    {
        "name": "AOSHeartbeat",
        "version": "1.07"
    },
    {
        "name": "AOSAlertManager",
        "version": "1.07"
    },
    {
        "name": "Family",
        "version": "1.0"
    },
    {
        "name": "FindMyMacMessenger",
        "version": "4.1"
    },
    {
        "name": "FollowUpUI",
        "version": "1.0"
    },
    {
        "name": "imagent",
        "version": "10.0"
    },
    {
        "name": "identityservicesd",
        "version": "10.0"
    },
    {
        "name": "IDSRemoteURLConnectionAgent",
        "version": "10.0"
    },
    {
        "name": "DiskImages UI Agent",
        "version": "559.40.3"
    },
    {
        "name": "AppSSOAgent",
        "version": "1.0"
    },
    {
        "name": "SpeechSynthesizerAuditor",
        "version": "1.0"
    },
    {
        "name": "ScreenReaderUIServer",
        "version": "10"
    },
    {
        "name": "KerberosMenuExtra",
        "version": "1.0"
    },
    {
        "name": "AquaAppearanceHelper",
        "version": "1.0"
    },
    {
        "name": "iCloudUserNotificationsd",
        "version": "1.0"
    },
    {
        "name": "SoftwareUpdateNotificationManager",
        "version": "1.0"
    },
    {
        "name": "VirtualScanner",
        "version": "4.1"
    },
    {
        "name": "PTPCamera",
        "version": "10.1"
    },
    {
        "name": "Type4Camera",
        "version": "10.1"
    },
    {
        "name": "MassStorageCamera",
        "version": "10.1"
    },
    {
        "name": "AirScanScanner",
        "version": "15"
    },
    {
        "name": "AutoImporter",
        "version": "6.7"
    },
    {
        "name": "MakePDF",
        "version": "10.1"
    },
    {
        "name": "Build Web Page",
        "version": "10.1"
    },
    {
        "name": "Python",
        "version": "2.7.16"
    },
    {
        "name": "Widget Simulator",
        "version": "1.0"
    },
    {
        "name": "SpeechSynthesisServer",
        "version": "9.0.24"
    },
    {
        "name": "FontRegistryUIAgent",
        "version": "81.0"
    },
    {
        "name": "AddressBookSync",
        "version": "11.0"
    },
    {
        "name": "AddressBookManager",
        "version": "11.0"
    },
    {
        "name": "AddressBookSourceSync",
        "version": "11.0"
    },
    {
        "name": "ABAssistantService",
        "version": "11.0"
    },
    {
        "name": "WebKitPluginHost",
        "version": "608"
    },
    {
        "name": "QuickLookUIHelper",
        "version": "5.0"
    },
    {
        "name": "Quick Look Simulator",
        "version": "1.0"
    },
    {
        "name": "Wish",
        "version": "8.5.9"
    },
    {
        "name": "SyncServer",
        "version": "8.1"
    },
    {
        "name": "PrinterProxy",
        "version": "15"
    },
    {
        "name": "ChineseTextConverterService",
        "version": "2.1"
    },
    {
        "name": "Summary Service",
        "version": "2.0"
    },
    {
        "name": "ImageCaptureService",
        "version": "6.7"
    },
    {
        "name": "Dictation",
        "version": "6.0.60"
    },
    {
        "name": "rcd",
        "version": "360"
    },
    {
        "name": "UIKitSystem",
        "version": "1.0"
    },
    {
        "name": "PowerChime",
        "version": "1.0"
    },
    {
        "name": "Script Menu",
        "version": "1"
    },
    {
        "name": "AppleScript Utility",
        "version": "1.1.2"
    },
    {
        "name": "ReportPanic",
        "version": "10.13"
    },
    {
        "name": "VoiceOver",
        "version": "10"
    },
    {
        "name": "CalendarFileHandler",
        "version": "8.0"
    },
    {
        "name": "AirPlayUIAgent",
        "version": "2.0"
    },
    {
        "name": "Paired Devices",
        "version": "1.9"
    },
    {
        "name": "AppleFileServer",
        "version": "2.1"
    },
    {
        "name": "Image Events",
        "version": "1.1.6"
    },
    {
        "name": "Jar Launcher",
        "version": "15.0.1"
    },
    {
        "name": "SpacesTouchBarAgent",
        "version": "1.0"
    },
    {
        "name": "TextInputSwitcher",
        "version": "1.1"
    },
    {
        "name": "ThermalTrap",
        "version": "1.0"
    },
    {
        "name": "SSMenuAgent",
        "version": "3.9.8"
    },
    {
        "name": "Keychain Circle Notification",
        "version": "1.0"
    },
    {
        "name": "NetAuthAgent",
        "version": "6.2"
    },
    {
        "name": "Dwell Control",
        "version": "1.0"
    },
    {
        "name": "AddressBookUrlForwarder",
        "version": "11.0"
    },
    {
        "name": "SocialPushAgent",
        "version": "62"
    },
    {
        "name": "WiFiAgent",
        "version": "13.0"
    },
    {
        "name": "DiscHelper",
        "version": "1.0"
    },
    {
        "name": "Install in Progress",
        "version": "3.0"
    },
    {
        "name": "OBEXAgent",
        "version": "7.0.3"
    },
    {
        "name": "TextInputMenuAgent",
        "version": "1.0"
    },
    {
        "name": "Automator Installer",
        "version": "2.10"
    },
    {
        "name": "iCloud",
        "version": "1.0"
    },
    {
        "name": "Java Web Start",
        "version": "15.0.1"
    },
    {
        "name": "Database Events",
        "version": "1.0.6"
    },
    {
        "name": "UserNotificationCenter",
        "version": "3.3.0"
    },
    {
        "name": "FolderActionsDispatcher",
        "version": "1.0"
    },
    {
        "name": "ODSAgent",
        "version": "1.8"
    },
    {
        "name": "RapportUIAgent",
        "version": "1.9"
    },
    {
        "name": "Google Chrome",
        "version": "80.0.3987.149"
    },
    {
        "name": "Endpoint Security for Mac Uninstaller",
        "version": "4.10.59.200059"
    },
    {
        "name": "EndpointSecurityforMac",
        "version": "4.10.59.200059"
    },
    {
        "name": "Spotify",
        "version": "1.1.26.501.gbe11e53b"
    },
    {
        "name": "zoom.us",
        "version": "4.6.7 (18176.0301)"
    },
    {
        "name": "Google Chrome Canary",
        "version": "83.0.4093.0"
    },
    {
        "name": "Microsoft To Do",
        "version": "2.13"
    },
    {
        "name": "Slack",
        "version": "4.3.3"
    },
    {
        "name": "Microsoft Outlook",
        "version": "16.35"
    },
    {
        "name": "Sourcetree",
        "version": "4.0.1"
    },
    {
        "name": "Visual Studio Code",
        "version": "1.43.1"
    },
    {
        "name": "Firefox",
        "version": "74.0"
    },
    {
        "name": "Tunnelblick",
        "version": "3.8.1 (build 5400)"
    },
    {
        "name": "CopyQ",
        "version": "3.8"
    },
    {
        "name": "Dropbox",
        "version": "93.4.273"
    },
    {
        "name": "iTerm",
        "version": "3.3.9"
    },
    {
        "name": "SecurePrint-HQ",
        "version": "15"
    },
    {
        "name": "Steam Helper",
        "version": "31.0.1650.57"
    },
    {
        "name": "Steam Helper EH",
        "version": "31.0.1650.57"
    },
    {
        "name": "Steam",
        "version": "1.5"
    },
    {
        "name": "360Daemon",
        "version": "1.0"
    },
    {
        "name": "FileZilla",
        "version": "3.44.2"
    },
    {
        "name": "Fax Receive Monitor",
        "version": "1.71"
    },
    {
        "name": "monitor",
        "version": "25.2.2"
    },
    {
        "name": "HP Device Monitor Manager",
        "version": "5.38.0"
    },
    {
        "name": "NetSpot",
        "version": "2.8.830"
    },
    {
        "name": "TeamViewer",
        "version": "15.3.2682"
    },
    {
        "name": "trivago Self Service",
        "version": "10.19.0"
    },
    {
        "name": "Sublime Text",
        "version": "Build 3211"
    },
    {
        "name": "HTML5App",
        "version": "1.0.0.0"
    },
    {
        "name": "Skype Meetings App",
        "version": "1.0.0.186"
    },
    {
        "name": "GIMP-2.10",
        "version": "2.10.14"
    },
    {
        "name": "ScanEventHandler",
        "version": "1.5.0"
    },
    {
        "name": "AppleMobileSync",
        "version": "5.0"
    },
    {
        "name": "Adobe Acrobat Reader DC",
        "version": "20.006.20042"
    },
    {
        "name": "Microsoft Word",
        "version": "16.35"
    },
    {
        "name": "Microsoft PowerPoint",
        "version": "16.35"
    },
    {
        "name": "Microsoft OneNote",
        "version": "16.35"
    },
    {
        "name": "Microsoft Excel",
        "version": "16.35"
    },
    {
        "name": "Microsoft AutoUpdate",
        "version": "4.21"
    },
    {
        "name": "Airtame",
        "version": "3.6.1"
    },
    {
        "name": "MobileDeviceUpdater",
        "version": "1.0"
    },
    {
        "name": "AppleMobileDeviceHelper",
        "version": "5.0"
    },
    {
        "name": "Firefox Developer Edition",
        "version": "75.0"
    },
    {
        "name": "HP DeskJet 3630 series",
        "version": "14.4"
    },
    {
        "name": "Xcode",
        "version": "11.3.1"
    },
    {
        "name": "Android Studio",
        "version": "3.5"
    },
    {
        "name": "Crash Reporter",
        "version": "1.0"
    },
    {
        "name": "calibre",
        "version": "3.31.0"
    },
    {
        "name": "MRT",
        "version": "1.56"
    },
    {
        "name": "Postman",
        "version": "7.19.1"
    },
    {
        "name": "AirScanLegacyDiscovery",
        "version": "15"
    },
    {
        "name": "Foxit Reader",
        "version": "3.4.0.1012"
    },
    {
        "name": "jamfHelper",
        "version": "10.19.0"
    },
    {
        "name": "Management Action",
        "version": "10.19.0"
    },
    {
        "name": "Acrobat Update Helper",
        "version": "1 . 0 . 18"
    },
    {
        "name": "DropboxMacUpdate",
        "version": "78.4.119"
    },
    {
        "name": "Safari",
        "version": "13.0.5"
    },
    {
        "name": "Skype",
        "version": "8.56"
    },
    {
        "name": "Skype for Business",
        "version": "16.28.142"
    },
    {
        "name": "Simulator",
        "version": "11.3.1"
    },
    {
        "name": "VirtualBox",
        "version": "5.2.28"
    },
    {
        "name": "ForkLift",
        "version": "3.1.2"
    },
    {
        "name": "Jamf",
        "version": "10.19.0"
    },
    {
        "name": "Keka",
        "version": "1.1.25"
    },
    {
        "name": "trivago",
        "version": "2020.1.7.25412"
    },
    {
        "name": "Cocoa-AppleScript Applet",
        "version": "1.0"
    },
    {
        "name": "rekordbox",
        "version": "5.6.0.0017"
    },
    {
        "name": "HP Scanner 3",
        "version": "4.22.1"
    },
    {
        "name": "HP Utility",
        "version": "5.38.0"
    },
    {
        "name": "Microsoft Edge Beta",
        "version": "79.0.309.43"
    },
    {
        "name": "Sourcetree",
        "version": "4.0"
    },
    {
        "name": "Opera",
        "version": "63.0"
    },
    {
        "name": "Chromium",
        "version": "80.0.3952.0"
    },
    {
        "name": "The Unarchiver",
        "version": "3.11.3"
    },
    {
        "name": "DBeaver",
        "version": "5.3.1"
    },
    {
        "name": "VLC",
        "version": "3.0.8"
    },
    {
        "name": "MySQLWorkbench",
        "version": "8.0.18.CE"
    },
    {
        "name": "Accessibility Inspector",
        "version": "5.0"
    },
    {
        "name": "FileMerge",
        "version": "2.11"
    },
    {
        "name": "Reality Composer",
        "version": "1.3"
    },
    {
        "name": "Instruments",
        "version": "11.3.1"
    },
    {
        "name": "Create ML",
        "version": "1.0"
    },
    {
        "name": "Flux",
        "version": "39.987"
    },
    {
        "name": "Conduktor",
        "version": "1.0-SNAPSHOT"
    },
    {
        "name": "glogg"
    },
    {
        "name": "Opera Installer",
        "version": "63.0.3368.94"
    },
    {
        "name": "SpeedCrunch",
        "version": "master"
    },
    {
        "name": "fax",
        "version": "5.28.0"
    },
    {
        "name": "FAX Utility",
        "version": "1.73"
    },
    {
        "name": "EPFaxAutoSetupTool",
        "version": "1.71"
    },
    {
        "name": "rastertoepfax",
        "version": "1.71"
    },
    {
        "name": "commandFilter",
        "version": "1.71"
    },
    {
        "name": "epsonfax",
        "version": "1.71"
    },
    {
        "name": "hpdot4d",
        "version": "5.4.0"
    },
    {
        "name": "hpdot4d",
        "version": "4.17.1"
    },
    {
        "name": "HP Product Research Manager",
        "version": "10.38.0"
    },
    {
        "name": "HP Fax Archive",
        "version": "5.38.0"
    },
    {
        "name": "Remove"
    },
    {
        "name": "Proof"
    },
    {
        "name": "Match"
    },
    {
        "name": "Embed"
    },
    {
        "name": "Extract"
    },
    {
        "name": "Show Info"
    },
    {
        "name": "Rename"
    },
    {
        "name": "Set Info"
    },
    {
        "name": "Canon IJScanner2",
        "version": "4.0.0"
    },
    {
        "name": "Canon IJScanner6",
        "version": "4.0.0"
    },
    {
        "name": "Canon IJScanner4",
        "version": "4.0.0"
    },
    {
        "name": "EPSON Scanner",
        "version": "5.7.24"
    },
    {
        "name": "pdftopdf",
        "version": "2.7.0"
    },
    {
        "name": "commandtohp",
        "version": "2.4.1"
    },
    {
        "name": "rastertofax",
        "version": "5.28.0"
    },
    {
        "name": "Xcode Server Builder",
        "version": "1.0"
    },
    {
        "name": "Docker Quickstart Terminal",
        "version": "1.3.0"
    },
    {
        "name": "Kitematic (Beta)",
        "version": "0.17.2"
    },
    {
        "name": "terminal-notifier",
        "version": "1.7.2"
    },
    {
        "name": "terminal-notifier",
        "version": "1.7.2"
    },
    {
        "name": "Python Launcher",
        "version": "2.7.15"
    },
    {
        "name": "Python Launcher 3",
        "version": "3.7.3"
    },
    {
        "name": "Python",
        "version": "3.7.3"
    },
    {
        "name": "Java Remove",
        "version": "Java Remove"
    },
    {
        "name": "Python",
        "version": "2.7.15"
    },
    {
        "name": "countinstalltion"
    },
    {
        "name": "Creative Cloud Installer",
        "version": "4.3.0.256"
    },
    {
        "name": "Java Mission Control",
        "version": "6.0.0"
    }
]
```

**Comment by somm-doe** [comment]
I am having the same issue - with other apps aswell though.

**Comment by pwstaten** [comment]
Also unable to enable driver. When I installed 360controller the first time, it worked (wired controller). A week or so later, the controller stopped working. In the preferences pane, I could not check "Enable Driver", without getting sent to the security and privacy pane. But the option to enable Drew Mills never showed up. (And I swear I already enabled Drew Mills when I first installed the software.) If I enable apps from anywhere (temporarily), checking the box still doesn't work, so I get the feeling that checking the box isn't even running (or attempting to run) any binary files at all.

**Comment by pwstaten** [comment]
Also, I ran spctl to check the notarization:

spctl -a -v --type install /Library/Extensions/360Controller.kext 
/Library/Extensions/360Controller.kext: accepted
source=Notarized Developer ID

And I tried force-quitting out of the preferences pain as FranticRain suggested, to no avail. So if something is wrong with maxrothman's mac, then something similar must be wrong with mine. (And this is with no USB peripherals.)

I suspect a system policy is preventing the kext from loading.
